### ISSUE TITLE
UI-Scale Setting (slider)

### ISSUE THREAD

**Issue by njalooo** [flxzt/rnote#1337]
**Is your feature request related to a problem? Please describe.**  
If one is using a display with an integrated graphic-tablet and touchscreen, the UI buttons are too small to reliably tap with your fingers.

**Describe the solution you'd like**  
I would like a setting, with which it is possible to increase (or decrease) the scale of the UI, ideally as a percentage slider. 

**Describe alternatives you've considered**  
Fixed size increments, in case of non-vectorized icons.

**Comment by flxzt** [comment]
This should be adjusted in your desktop environment (Gnome for example recently added the option for fractional scaling).

**Comment by njalooo** [comment]
@flxzt 
I disagree, except for rnote, i am using my trackpad for operating my laptop, and for that my regular scale (which is on the smaler side) suits me just fine, since i prefer the additional amount of information it can display. Having to adjust my global scaling everytime i use rnote is simply not practical. 
The issue arises because using a touchscreen is less precise than a trackpad, so it needs bigger elements in order to use it reliably. Since rnote is a program where it makes a lot of sense to use the touchscreen in conjunction with a stylus, i think that it should have it's own scaling setting, that scales off the global scale.

**Comment by pyrotato** [comment]
Agreeing w/ @njalooo here (apologies if this resurrects the issue or something, I'm not very familiar w/ how issues work + etiquette)

I'm on a smaller laptop using just a drawing tablet to interact w/ rnote, and the buttons are small + hard to read and target. Every other application is scaled fine for me except for this one, so changing up my entire environment just for this application throws off pretty much everything else. It'd be nice to have scalable UI just to find that nice balance between document space and buttons.

Other than that, rnote's been great to use so far! This would just make for a nice addition on top of everything haha
